### PR TITLE
Set better expectations about technical documentation

### PR DESCRIPTION
**Pull request by anikbrazeau**
# Summary | Résumé

Rename documentation to API documentation to clarify what to expect

### CODE REVIEW

**Comment by anikbrazeau** [comment]
[Content updates to clarify trail mode](https://trello.com/c/N5fK1VMH/390-content-updates-to-clarify-trail-mode)

File: app/translations/csv/fr.csv [hunk 1]
```diff
@@ -1285,7 +1285,7 @@
 "Your request to go live is being reviewed. We’ll be in touch within 2 business days.","Votre demande d’activation est en cours. Nous communiquerons avec vous dans un délai de 2 jours ouvrables."
 "Go Live request for {}","Demande d’activation pour {}"
 "Your request was submitted.","Votre demande a été soumise."
-"Integrating the API into your system allows you to automate notifications and send emails with attachments. Find out more in the <a href='{}'> documentation for developers</a>.","Vous pouvez intégrer l’API dans votre système pour automatiser les notifications et envoyer des courriels avec pièces jointes. Pour en savoir plus, consultez la  <a href='{}'>documentation pour les développeurs</a>."
+"Integrating the API into your system allows you to automate notifications and send emails with attachments. Find out more in the <a href='{}'> API documentation</a>.","Vous pouvez intégrer l’API dans votre système pour automatiser les notifications et envoyer des courriels avec pièces jointes. Pour en savoir plus, consultez la  <a href='{}'>documentation API</a>."
```
Comment (AntoineAugusti) [inline app/translations/csv/fr.csv:1288]:
```suggestion
"Integrating the API into your system allows you to automate notifications and send emails with attachments. Find out more in the <a href='{}'>API documentation</a>.","Vous pouvez intégrer l’API dans votre système pour automatiser les notifications et envoyer des courriels avec pièces jointes. Pour en savoir plus, consultez la  <a href='{}'>documentation API</a>."
```

File: app/templates/views/api/index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h1 class="heading-medium mb-12 clear-both contain-floats">
 
   <div class="mb-12">
     <p>
-      {{ _("Integrating the API into your system allows you to automate notifications and send emails with attachments. Find out more in the <a href='{}'> documentation for developers</a>.").format(url_for('main.documentation')) }}
+      {{ _("Integrating the API into your system allows you to automate notifications and send emails with attachments. Find out more in the <a href='{}'> API documentation</a>.").format(url_for('main.documentation')) }}
```
Comment (AntoineAugusti) [inline app/templates/views/api/index.html:17]:
```suggestion
      {{ _("Integrating the API into your system allows you to automate notifications and send emails with attachments. Find out more in the <a href='{}'>API documentation</a>.").format(url_for('main.documentation')) }}
```

**Review by AntoineAugusti — APPROVED** [review]
Removed an extra space in the link, otherwise 🚢

**Pull request merged**